src/ontology/mondo-ingest.Makefile

```diff
@@ -29,7 +29,8 @@ $(TMPDIR)/ordo_relevant_signature.txt: component-download-ordo.owl | $(TMPDIR)
 
 # This section is concerned with transforming the incoming sources into the
 # Monarch Ingest schema.
-# https://github.com/monarch-initiative/omim/issues/60
+# Illegal punning on some properties #60: https://github.com/monarch-initiative/omim/issues/60
```

Comment:
@matentzn There was a comment w/ a link to [Illegal punning on some properties #60](https://github.com/monarch-initiative/omim/issues/60) here on the `$(COMPONENTSDIR)/omim.owl` goal. I don't remember if I added this, or you. Is this because the illegal punning is being fixed here in `mondo-ingest` but it should eventually be fixed in OMIM?